Userland/Libraries/LibGUI/Frame.h

```diff
@@ -23,6 +23,7 @@ class Frame : public Widget {
 
     Gfx::FrameStyle frame_style() const { return m_style; }
     void set_frame_style(Gfx::FrameStyle);
+    inline void set_style(Gfx::FrameStyle style) { set_frame_style(style); }
```

Comment:
It would probably be more consistent/straightforward to replace the use of `style` in the GML file with `frame_style`. As far as I'm aware, that's already something that is an entry on a few peoples' TODO lists anyways.